src/Chapter3.hs

```diff
@@ -907,9 +1023,34 @@ Implement instances of "Append" for the following types:
   ✧ *(Challenge): "Maybe" where append is appending of values inside "Just" constructors
 
 -}
+newtype Gold = Gold { goldValue :: Int }
 class Append a where
-    append :: a -> a -> a
+	append :: a -> a -> a
+
+instance Append Gold where
+	append :: Gold -> Gold -> Gold
+	append x1 x2 = Gold ((goldValue x1) + (goldValue x2))
 
+instance Append [a] where
+	append :: [a] -> [a] -> [a]
+	append x1 x2 = x1 ++ x2
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
      append = (++)
```